staging/src/kubevirt.io/client-go/go.mod

```diff
@@ -52,7 +52,7 @@ require (
 	github.com/pkg/errors v0.9.1 // indirect
 	github.com/x448/float16 v0.8.4 // indirect
 	golang.org/x/net v0.36.0 // indirect
-	golang.org/x/oauth2 v0.23.0 // indirect
+	golang.org/x/oauth2 v0.27.0 // indirect
```

Comment:
Interesting I would expect that workspace would sync this from root `go.mod`. @0xFelix wdyt?

Reply:
You need to run `go work sync`, I don't think it syncs automatically. We run it in our dep scripts though.